pkg/systemd/host.js

```diff
@@ -228,15 +229,19 @@ PageServer.prototype = {
             $elt.show();
 
             var model = {
-                Synched: self.server_time.timedate.NTPSynchronized
+                Synched: self.server_time.timedate.NTPSynchronized,
+                service: "ntpd.service"
```

Comment:
Why is there an assumption that ntpd.service will be in use when timesyncd is not? What about chronyd? Shouldn't we be treating all non timesyncd as one case?